.github/workflows/main.yml

```diff
@@ -54,12 +55,21 @@ jobs:
       - name: Pack
         run: dotnet publish -c Release /p:Version=${{ env.VERSION }} ./source
 
+      - name: Login to Octopus Deploy 🐙
+        if: (! contains(github.ref, '/merge')) && (! contains(github.ref, '/dependabot/')) && (! contains(github.ref, 'prettybot/'))
+        uses: OctopusDeploy/login@v1
+        with: 
+          server: https://deploy.octopus.app
+          service_account_id: 2552cea7-8cee-454e-a56b-e20e373e9c1f
+
       - name: Push to Octopus 🐙
+        if: (! contains(github.ref, '/merge')) && (! contains(github.ref, '/dependabot/')) && (! contains(github.ref, 'prettybot/'))
```

Comment:
This is an intentional change to bring it inline with other repos where we don't push to octopus on PR validation build